homeassistant/components/zwave_js/const.py

```diff
@@ -33,3 +33,10 @@
 ATTR_PROPERTY_NAME = "property_name"
 ATTR_PROPERTY_KEY_NAME = "property_key_name"
 ATTR_PARAMETERS = "parameters"
+
+# service constants
+SERVICE_SET_CONFIG_VALUE = "set_config_value"
```

Comment:
```suggestion
SERVICE_SET_CONFIG_VALUE = "set_config_parameter"
```
This is consistent with the zwave and ozw integrations, and "parameter" is the common terminology.

Reply:
I'm not sure about this. I think "set_config_value" better suits here.

Reply:
Why? It's suited only the the terminology of the API. "parameter" is commmon; it's used everywhere, in OZW, in zwave-js docs and code, **in the spec**, etc.

Reply:
as a non-english speaking user "parameter" sounds a bit strange, that's all. But I'm fine with parameter or param